image.yaml

```diff
@@ -67,6 +67,9 @@ modules:
   - path: modules
   install:
   - name: org.infinispan.dnf-workaround
+    # Jdk must be installed before other dependencies to allow for cached dnf files to be removed
```

Comment:
@ryanemerson Didn't know about this, maybe it's the same for OpenJ9, too? If that's the case, it would be nice to change the order on the `openj9-override.yaml` file, too. If you prefer, I can issue a PR right after this one is merged to do that, or you can just include the change in this one - it's up to you.